__tests__/standup/standup.test.js

```diff
@@ -79,4 +79,16 @@ describe('Input box', () => {
     const loader = await page.$('.loader');
     expect(loader).toBeTruthy();
   });
+
+  it('should update the URL with the query parameter when the user writes a name', async () => {
```

Comment:
any tests to verify multiple query params?

Reply:
Not needed

Reply:
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/70854507/8a13f0c6-c187-45d4-9b46-f0796255e7d2)